tailord/src/dbus/profiles.rs

```diff
@@ -94,6 +122,16 @@ impl ProfileInterface {
                 .map_err(|err| fdo::Error::Failed(err.to_string()))?;
         }
 
+        if let Some(perf_handle) = &self.performance_profile_handle {
+            if let Some(performance_profile) = performance_profile {
+                perf_handle
+                    .profile_sender
+                    .send(performance_profile.to_string())
+                    .await
+                    .map_err(|err| fdo::Error::Failed(err.to_string()))?;
+            }
```

Comment:
Can't you simply update the value of `PerformanceProfileRuntimeHandle` here? This way, we could always return the actually loaded performance profile name.